app/models/manageiq/providers/cloud_manager/provision_workflow/dialog_field_validation.rb

```diff
@@ -2,7 +2,14 @@
 module ManageIQ::Providers::CloudManager::ProvisionWorkflow::DialogFieldValidation
   def validate_cloud_subnet(field, values, dlg, fld, value)
     return nil unless value.blank?
-    return nil if get_value(values[:cloud_network]).to_i.zero?
+    return nil if get_value(values[:placement_auto]) == true
```

Comment:
You don't need the `== true` and I believe `get_option(:placement_auto)` should work so try
```suggestion
    return nil if get_option(:placement_auto)
```

Reply:
Actually I see a `#placement_auto` method, can you try `return nil if placement_auto` works?

Reply:
Yeah I'll give it a try.

Reply:
Changed to just removing the `== true` since it seems like the placement_auto function is undefined in app/models/manageiq/providers/cloud_manager/provision_workflow/dialog_field_validation.rb